.travis.yml

```diff
@@ -48,9 +48,6 @@ jobs:
       after_success:
         - .travis/codecoverage.sh
 
-    - env: NAME=NODE_11
```

Comment:
We shouldn't remove this yet, Node 11 is still a supported platform (until 2019-06-01).

Reply:
done